pkg/remotewrite/config.go

```diff
@@ -41,6 +41,12 @@ type Config struct {
 	// Password is the Password for the Basic Auth.
 	Password null.String `json:"password"`
 
+	// ClientCertificate is the public key of the SSL certificate.
```

Comment:
Here (and the same comment for key's key), I believe worth mentioning that the path is expected, not the data

Reply:
Done